openvpn/files/common_opts.jinja

```diff
@@ -4,6 +4,10 @@
 dev-node {{ config.dev_node }}
 {%- endif %}
 
+{%- if config.dev_type is defined %}
```

Comment:
Please document `dev_type` in `pillar.example`.

Reply:
done. also included the other new options

Reply:
Thank you very much!